fs/littlefs/Make.defs

```diff
@@ -41,7 +41,7 @@ CFLAGS += -DLFS_NAME_MAX=$(CONFIG_FS_LITTLEFS_NAME_MAX)
 CFLAGS += -DLFS_FILE_MAX=$(CONFIG_FS_LITTLEFS_FILE_MAX)
 CFLAGS += -DLFS_ATTR_MAX=$(CONFIG_FS_LITTLEFS_ATTR_MAX)
 
-LITTLEFS_VERSION ?= 2.4.0
+LITTLEFS_VERSION ?= 2.5.1
```

Comment:
why did you bump littlefs version in this PR?

Reply:
@radek-pesina could you please answer this?

Reply:
The logic behind the bump was simply to ensure any more recent bugfixes and improvements to littlefs were incorporated into our use.